config.yaml.sample

```diff
@@ -51,19 +51,6 @@ clair:
     keyfile:
     certfile:
 
```

Comment:
User won't be able to define the detectors any more. However, user may want to do the filtering on API level, which means that *maybe* we should change the API to allow the user to query Clair with a set of feature/namespace detectors.